run

```diff
@@ -96,6 +96,8 @@ options:
     -f, --force                    stop running VM
     -c, --clean                    remove kernel and initramfs volumes after VM
                                    shutdown
+    --shared                       using scsi disks as shared disk (The qcow2
```

Comment:
It wasn't clear from the PR why qcow2 can't be used for shared disk, but the ticket somewhat explains that:
https://bugzilla.redhat.com/show_bug.cgi?id=1511480